src/app/loadout-builder/PerkPicker.tsx

```diff
@@ -31,6 +36,10 @@ import { escapeRegExp } from 'app/search/search-filters';
 import { D2ManifestDefinitions } from 'app/destiny2/d2-definitions';
 import { SocketDetailsMod } from 'app/item-popup/SocketDetails';
 
+const alreadyInsertedRule: DestinyPlugRuleDefinition = {
+  failureMessage: 'Similar mod already applied.'
```

Comment:
is this i18n-proof?

![image](https://user-images.githubusercontent.com/31990469/69913308-ff04c880-13ea-11ea-8646-9eed712c4249.png)
![image](https://user-images.githubusercontent.com/31990469/69913309-01ffb900-13eb-11ea-8389-3924b71e77b9.png)

Reply:
Not by the looks of what you have posted here. This is the part I didn't like when I referred to this PR in my slack comment.